src/docfx/lib/markdown/MarkdownVisitContext.cs

```diff
@@ -16,9 +18,12 @@ internal class MarkdownVisitContext
 
         public bool IsInclude => FileStack.Count >= 2;
 
+        public MonikerList ZoneMoniker => ZoneMonikerStack.Count >= 1 ? ZoneMonikerStack.Peek() : default;
```

Comment:
nit: here is now a `TryPeak` method

```suggestion
        public MonikerList ZoneMoniker => ZoneMonikerStack.TryPeek(out var top) ? top : default;
```